src/Executables/Auth/PacketHandlers/LoginRequestHandler.cs

```diff
@@ -32,7 +55,7 @@ public async Task ExecuteAsync(AuthPacketContext<LoginRequest> ctx, Cancellation
             _logger.LogDebug("Account {Username} not found", ctx.Packet.Username);
             ctx.Connection.Send(new LoginFailed
             {
-                Status = "WRONGPWD"
+                Status = LoginFailedBecause.NoId.ToString().ToUpper()
```

Comment:
Please use [`Enums.NET`](https://github.com/TylerBrinkley/Enums.NET) for converting enums to string. It's already installed in the Game project.
Should be as simple as `LoginFailedBecause.WrongPwd.AsString()`

Also please do not return different error messages when the account could not be found. This is a security issue. Do not give attackers more info than needed.